OrbitCore/FramePointerValidatorTests.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "ElfFile.h"
+#include "FramePointerValidator.h"
+#include "OrbitFunction.h"
+#include "Path.h"
+
+namespace {
```

Comment:
I don't remember the reason but I think we had a discussion some time ago about the fact that there's no need to put tests inside the anonymous namespace.